modules/admin/proc-generating-postgresql-certificate-for-aws-rds.adoc

```diff
@@ -0,0 +1,36 @@
+[id='generating-postgresql-certificate-for-aws-rds_{context}']
+= Generating PostgreSQL certificate for AWS RDS
+
+For Amazon Web Services (AWS) Relational Database Service (RDS), you can generate a PostgreSQL certificate file in the .pem format. The content of this file is used to create a PostgreSQL secret for {product-short} configuration.
```

Comment:
No need to generate anything for RDS, just download a certificate bundle (as mentioned below), it is in PEM already.